src/components/ButtonAuth.jsx

```diff
@@ -0,0 +1,29 @@
+import Image from "next/image";
+import { Button } from "react-flatifycss";
+
+import "flatifycss/dist/css/flatify-min.css";
+import styles from "./ButtonAuth.module.css";
+
+const ButtonAuth = (props) => {
+    return (
+        <>
+            <Button
+                className={`w-full h-14 px-6 transition ${styles.ButtonAuth}`}
+            >
+                <div className='relative flex items-center space-x-4 justify-center'>
+                    <Image
+                        src={props.icon}
+                        alt={props.alt}
+                        width={props.width}
+                        height={props.height}
+                    />
```

Comment:
@SufyanCS Great work! I have a little suggestion here to use destruct these values and use them instead of typing `props` every time.
```javascript
const { icon, alt, width, height } = props // at the start of the component.
```